nss-day-cohort-44/rare-rest-api-malicous-vortices
# Reactions

# ISSUE TICKET 
## Description
Should be able to add reactiions, list them, and add post reactions

## Type of change

# Testing Instructions
`git fetch origin `
`git checkout ` 

To list reactions: 

http://localhost:8000/reactions

to delete, pick a primary key and use a delete to http://localhost:8000/reactions/(whatever)

to create a post reaction, use a post method and send a request to: 
http://localhost:8000/reactions/(primary key of reaction you want to use)/react

This post must have a body of {"post" : (primary key of post I am reacting to)}

delete works with the same URL and must also have a body with the post primary key you're reacting to.

Check in your database to see that post reactions have been created/deleted.
User Instructions: 

# Checklist:
